packages/next/src/server/config-schema.ts

```diff
@@ -198,11 +198,7 @@ export const configSchema: zod.ZodType<NextConfig> = z.lazy(() =>
     compress: z.boolean().optional(),
     configOrigin: z.string().optional(),
     crossOrigin: z
-      .union([
-        z.literal(false),
-        z.literal('anonymous'),
-        z.literal('use-credentials'),
-      ])
+      .union([z.literal('anonymous'), z.literal('use-credentials')])
```

Comment:
Isn't that a breaking change?

Reply:
It's a bugfix, but regardless this option wasn't actually being used, it wasn't even documented actually, we're adding docs today 🙏